test-it/ViaTests/TestVI_GetVariantAttribute_IncompatibleClusterDataTypeReportsError.via

```diff
@@ -0,0 +1,85 @@
+// Autogenerated Vireo assembly file
+// Generated on 03/26/2019 16:39:18
+//---------------------------------------------------
+//VI Definition: ::Web Server::Interactive::Application::TestVI_GetVariantAttribute_IncompatibleClusterDataTypeReportsError.gviweb
```

Comment:
![TestVI_GetVariantAttribute_IncompatibleClusterDataTypeReportsError](https://user-images.githubusercontent.com/2609749/55037958-57c6ca80-4fed-11e9-8759-2141997d341d.PNG)

Reply:
Updated this second cluster (Cluster(Bool+Double)) to instead be a Cluster(Bool). That way, the two clusters are not coercible and will fail on desktop too.